### PR TITLE
Add method for new table creation

### PR DESCRIPTION
**Pull request by charulagrl**

### CODE REVIEW

File: createdb.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+import os
+import pkg_resources
+import fedmsg.config
+
+from summershum.model import create_tables
+
+basedir = os.path.abspath(os.path.dirname(__file__))
+
+create_tables(fedmsg.config.get('summershum.sqlalchemy.url') or 
+	'sqlite:///' + os.path.join(basedir, 'summershum.db'), True)
```
Comment (pypingou) [inline createdb.py:14]: I would define the db_url above and use it in the `config.get()` instead

File: summershum/model.py [hunk 2]
```diff
@@ -7,6 +7,39 @@
...
+    if db_url.startswith('sqlite:'):
+        # Ignore the warning about con_record
+        # pylint: disable=W0613
+        def _fk_pragma_on_connect(dbapi_con, con_record):
+            ''' Tries to enforce referential constraints on sqlite. '''
+            dbapi_con.execute('pragma foreign_keys=ON')
+        sa.event.listen(engine, 'connect', _fk_pragma_on_connect)
+
+    # Generate the Alembic version table and "stamp" it with the latest rev
+    alembic.command.stamp(get_alembic_config(db_url), "head")
```
Comment (pypingou) [inline summershum/model.py:37]: Where is this coming from?

Comment (charulagrl) [inline summershum/model.py:37]: @pypingou  This is from https://github.com/fedora-infra/pkgdb2/blob/master/pkgdb2/lib/model/__init__.py#L69. This was just for experimentation, later on I will make changes in it.

Comment (pypingou) [inline summershum/model.py:37]: As is I doubt this works :-)

**Comment by pypingou** [comment]
@charulagrl ping?

**Comment by ralphbean** [comment]
OK, we haven't heard back on this so I'm going to close it.  Please resubmit if there's ongoing work.

**Pull request closed without merging**